perfkitbenchmarker/linux_benchmarks/cloudsuite_web_search_benchmark.py

```diff
@@ -0,0 +1,195 @@
+"""
+
+DISK_PATH = '/scratch'
+
+
+def GetConfig(user_config):
+  config = configs.LoadConfig(BENCHMARK_CONFIG, user_config, BENCHMARK_NAME)
+  if FLAGS['num_vms'].present:
+    config['vm_groups']['clients']['vm_count'] = FLAGS.num_vms
+  return config
+
+
+def Prepare(benchmark_spec):
+  """Install docker. Pull the required images from DockerHub.
+
+  Start Solr index node and client.
+
+  Args:
+    benchmark_spec: The benchmark specification. Contains all data that is
+        required to run the benchmark.
+  """
+  servers = benchmark_spec.vm_groups['servers'][0]
+  clients = benchmark_spec.vm_groups['clients']
+
+  def PrepareCommon(vm):
+    if not docker.IsInstalled(vm):
+      vm.Install('docker')
+
+  def PrepareServer(vm):
+    PrepareCommon(vm)
+    server_cmd = ('sudo echo \'DOCKER_OPTS="-g %s"\''
+                  '| sudo tee /etc/default/docker > /dev/null' % (DISK_PATH))
+    stdout, _ = vm.RemoteCommand(server_cmd, should_log=True)
+
+    server_cmd = 'sudo service docker restart'
+    stdout, _ = vm.RemoteCommand(server_cmd, should_log=True)
+
+    vm.RemoteCommand('sudo docker pull cloudsuite/web-search:server')
+
+    server_cmd = ('sudo docker run -d --net host '
+                  '--name server cloudsuite/web-search:server %s 1' %
+                  (FLAGS.cloudsuite_web_search_server_heap_size))
+
+    stdout, _ = servers.RemoteCommand(server_cmd, should_log=True)
+
+  def PrepareClient(vm):
+    PrepareCommon(vm)
+    vm.RemoteCommand('sudo docker pull cloudsuite/web-search:client')
+    time.sleep(120)
+
+  target_arg_tuples = ([(PrepareClient, [vm], {}) for vm in clients] +
+                       [(PrepareServer, [servers], {})])
+  vm_util.RunParallelThreads(target_arg_tuples, len(target_arg_tuples))
+
+
+def Run(benchmark_spec):
+  """Run the Web Search benchmark.
+
+  Args:
+    benchmark_spec: The benchmark specification. Contains all data that is
+        required to run the benchmark.
+
+  Returns:
+    A list of sample.Sample objects.
+  """
+  clients = benchmark_spec.vm_groups['clients'][0]
+  servers = benchmark_spec.vm_groups['servers'][0]
+  results = []
+
+  benchmark_cmd = ('sudo docker run --rm --net host --name client '
+                   'cloudsuite/web-search:client %s %d %d %d %d ' %
+                   (servers.internal_ip,
+                    FLAGS.cloudsuite_web_search_scale,
+                    FLAGS.cloudsuite_web_search_ramp_up,
+                    FLAGS.cloudsuite_web_search_steady_state,
+                    FLAGS.cloudsuite_web_search_ramp_down))
+  stdout, _ = clients.RemoteCommand(benchmark_cmd, should_log=True)
+
+  ops_per_sec = re.findall(r'\<metric unit="ops/sec"\>(\d+\.?\d*)', stdout)
+  sum_ops_per_sec = float(ops_per_sec[0])
+  p90 = re.findall(r'\<p90th\>(\d+\.?\d*)', stdout)
+  sum_p90 = float(p90[0])
+  p99 = re.findall(r'\<p99th\>(\d+\.?\d*)', stdout)
+  sum_p99 = float(p99[0])
+
+  results = []
+  results.append(sample.Sample('Operations per second', sum_ops_per_sec,
```

Comment:
Could you remove or rename the `sum_` variables? Nothing is being summed anymore.